ui/00_message/pawn/p0001.xml

```diff
@@ -17,22 +19,28 @@
   </message>
   <message>
     <original>冒険の手助けなら、いつでもいたします</original>
+	<translation>If you are in need of help on your adventure, we are here for you.</translation>
   </message>
   <message>
     <original>まだ、作業を承っておりません</original>
+	<translation>I have not recieved work yet.</translation>
   </message>
   <message>
     <original>ご指示をお待ちしています</original>
+	<translation>We are awaiting instructions.</translation>
   </message>
   <message>
     <original>旅かクラフトのご用命あらば
 いつでも動きます</original>
+	<translation>We will follow you in your travels. We can move at anytime.</translation>
```

Comment:
I don't know how close we're supposed to generally stick to the original text, but, "(Whether it's) crafting or traveling, if you have an order, I'll follow it any time."